04_numeronym/NimishMudhale/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func numeronym(vals ...string) []string {
+	numeronymArray := make([]string, len(vals))
+	for i, val := range vals {
+		target := strings.Trim(val, "\n\t")
+		len := len(target)
+		if len <= 3 {
+			numeronymArray[i] = target
+			continue
```

Comment:
Conceptually you should put the `else` case in an `else` block.
`continue` should be used more as a blocker to skip remaining steps than to facilitate `if`-`else` logic, even though they are logically identical in this case.

Reply:
done